about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
+  <link rel="stylesheet" href="about.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="show-case">
+    <div class="overlay"></div>
+    <header>
+      <div class="social">
+        <ul class="social-links d-flex">
+          <li><a href="#"><i class='bx bxl-facebook-circle bx-sm'></i></a></li>
+          <li><a href="#"><i class='bx bxl-twitter bx-sm'></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <nav class="d-flex">
+        <a class="burger" href="#">
+          <i class='bx bx-menu bx-md'></i>
+        </a>
+        <a href="index.html">
+          <h1 class="frank-logo" >Musica</h1>
+        </a>
+        <ul class="desktop-links">
+          <li class="primary-color"><a href="about.html">About</a></li>
+          <li ><a class="secondary-color" href="#">Program</a></li>
+          <li ><a class="secondary-color" href="#">Join</a></li>
+          <li><a class="secondary-color" href="#">Sponsor</a></li>
+          <li><a class="secondary-color" href="#">News</a></li>
+          <li class="primary-color spec"><a href="#">Campaign</a></li>
+        </ul>
+      </nav>
+      <div class="menu-container">
+        <div class="close-btn">
+          <i class="bx bx-x bx-lg closeIcon"></i>
+        </div>
+        <ul>
+          <li><a class="side-link" href="index.html">Home</a></li>
+          <li><a class="side-link" href="about.html">About</a></li>
+          <li><a class="side-link" href="#aboutme">Program</a></li>
+          <li><a class="side-link" href="#contact">Join</a></li>
+          <li><a class="side-link" href="#contact">Sponsor</a></li>
+          <li><a class="side-link" href="#contact">Campaign</a></li>
```

Comment:
- [ ] The mobile menu also, remove the about page link. and re-adjust the margin of the links and the `X` close button to the far right.

![m](https://user-images.githubusercontent.com/61562626/142412241-3f2e9565-6b6a-48bc-8874-8f09970f3b63.jpeg)